src/test/java/space/obminyashka/items_exchange/end2end/LocationFlowTest.java

```diff
@@ -99,7 +99,51 @@ private void sendAndCompareLocationResponse(MockHttpServletRequestBuilder reques
                 .andExpect(jsonPath("$.areaUA").value(AREA_UA))
                 .andReturn();
     }
-
+    @WithMockUser(username = "admin", roles = { "ADMIN" })
+    @Test
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "location/create.yml", orderBy = "city_en", ignoreCols = "id")
+    void createTwoSameLocation_shouldGetMessageBadRequest() throws Exception
+    {
+        sendAndCompareLocationResponse(post(LOCATION), status().isCreated());
+        sendDtoAndGetResultAction(post(LOCATION), createValidLocationDto(), status().isBadRequest());
```

Comment:
It would be much better to use a new custom dataset instead of making 2 requests in a test.